tests/integration/components/polaris-data-table-test.js

```diff
@@ -80,11 +82,14 @@ module('Integration | Component | polaris-data-table', function(hooks) {
         sortable=firstColumnSortable
       }}
     `);
-    const firstHeadingCell = findAll('th.Polaris-DataTable__Cell')[0];
 
-    assert.ok(
-      firstHeadingCell.classList.contains('Polaris-DataTable__Cell--sorted')
-    );
+    const firstHeadingCell = findAll(
+      '[data-test-data-table-cell][data-test-data-table-cell-header]'
+    )[0];
+
+    assert
+      .dom(firstHeadingCell)
+      .hasAttribute('data-test-data-table-cell-sorted', 'true');
```

Comment:
hmm, I don't think we should use selectors like this....we should use selectors just to match elements...in this case we just that the element has a test selector which is not what we want

here, for example, we should test that it has the sorted class....we could remove that class and the test would still pass, but functionality broken no?

Reply:
same in a bunch of other places around here

Reply:
Yeah I'm totally fine with testing that it has the sorted class, I just wasn't sure how hard we were enforcing the use of test-selectors so I was trying to use them for all cases.